src/ProductConstructionService/ProductConstructionService.Api/Telemetry/TelemetryRecorder.cs

```diff
@@ -3,25 +3,33 @@
 
 using System.Diagnostics;
 using Microsoft.ApplicationInsights;
-using ProductConstructionService.Api.Queue.Jobs;
+using Microsoft.DotNet.DarcLib;
 
 namespace ProductConstructionService.Api.Telemetry;
 
-public class TelemetryRecorder(ILogger<TelemetryRecorder> logger, TelemetryClient telemetryClient) : ITelemetryRecorder
+public class TelemetryRecorder(
+        ILogger<TelemetryRecorder> logger,
+        TelemetryClient telemetryClient)
+    : ITelemetryRecorder
 {
     private readonly ILogger<TelemetryRecorder> _logger = logger;
     private readonly TelemetryClient _telemetryClient = telemetryClient;
 
-    public ITelemetryScope RecordJob(Job job)
+    public ITelemetryScope RecordJob(string jobName)
     {
-        return new TelemetryScope($"JobExecuted", _logger, _telemetryClient, new() { ["JobType"] = job.Type }, []);
+        return new TelemetryScope($"JobExecuted", _logger, _telemetryClient, new() { ["JobType"] = jobName }, []);
```

Comment:
We're a bit inconsistent here with the `JobType`, `jobName`. These should probably match

Reply:
Good point